src/o365/yammer/commands/message/message-add.ts

```diff
@@ -0,0 +1,164 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  body: string;
+  repliedToId?: number;
+  directToUserIds?: string;
+  groupId?: number;
+  networkId?: number;
+}
+
+class YammerMessageAddCommand extends YammerCommand {
+  public get name(): string {
+    return `${commands.YAMMER_MESSAGE_ADD}`;
+  }
+
+  public get description(): string {
+    return 'Posts a Yammer network message on behalf of the current user';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.repliedToId = args.options.repliedToId !== undefined;
+    telemetryProps.directToUserIds = args.options.directToUserIds;
```

Comment:
I changed this to `telemetryProps.directToUserIds = args.options.directToUserIds !== undefined;` since this property can contain emails and we should not log this in the telemetry.